// Reversed by pbd (https://github.com/rsc-dev/pbd)
package Devialet.GetThePartyStarted;
import "CallMeMaybe/CommonMessages.proto";

message UploadLogsRequest {
	required string installationId = 1 ;
	required int64 timestamp = 2 ;
}

service LogUploader {
	rpc uploadLogs(.Devialet.GetThePartyStarted.UploadLogsRequest) returns (.Devialet.CallMeMaybe.Empty) {
	}
	option (dvltServiceOptions) = {
		serviceName: "com.devialet.getthepartystarted.log-uploader-0"
		
	};
}